clients/python/src/model_registry/_client.py

```diff
@@ -72,7 +72,7 @@
 logger = logging.getLogger("model-registry")
 
 DEFAULT_USER_TOKEN_ENVVAR = "KF_PIPELINES_SA_TOKEN_PATH"  # noqa: S105
-DEFAULT_K8S_SA_TOKEN_PATH = "/var/run/secrets/kubernetes.io/serviceaccount/token" # noqa: S105
+DEFAULT_K8S_SA_TOKEN_PATH = "/var/run/secrets/kubernetes.io/serviceaccount/token"  # noqa: S105
```

Comment:
```suggestion
DEFAULT_K8S_SA_TOKEN_PATH = "/var/run/secrets/kubernetes.io/serviceaccount/token" # noqa: S105
```

not sure I get this modification? 🙃

Reply:
I ran `ruff` on this file which mostly formatted my code changes, but affected a couple other lines. I updated the PR to revert this and other formatting outside of my code, since we don't have make commands and CI maintaining this currently.

Reply:
ah, that's weird of Ruff 🤔